operations/maintainer.go

```diff
@@ -0,0 +1,89 @@
+package operations
+
+import (
+	"context"
+	"github.com/Peripli/service-manager/pkg/log"
+	"github.com/Peripli/service-manager/pkg/query"
+	"github.com/Peripli/service-manager/pkg/types"
+	"github.com/Peripli/service-manager/pkg/util"
+	"github.com/Peripli/service-manager/storage"
+	"time"
+)
+
+// OperationMaintainer ensures that operations old enough are deleted
+// and that no stuck (orphan) operations are left in the DB due to crashes/restarts of SM
+type OperationMaintainer struct {
+	repository      storage.Repository
```

Comment:
Use transactional repository here and wrap the Execute in InTransaction

Reply:
Discussed that it's not necessary.